src/samples/OrientedBoundingBox.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Rock-Core
+#ifndef BASE_SAMPLES_ORIENTEDBOUNDINGBOX_HPP_
+#define BASE_SAMPLES_ORIENTEDBOUNDINGBOX_HPP_
+
+#include <base/samples/BoundingBox.hpp>
+
+namespace base {
+namespace samples {
+/**
+ * Oriented bounding box (OBB) representation with associated uncertainty
+ * information.
```

Comment:
Please document the semantics of the rotation. Is it applied in whatever frame the box itself is defined ? Is it a rotation around the center position ? (I'm guessing the latter, but please document it)

Reply:
It is a rotation around the center position. I am going to fix it.